apps/contrib/templatetags/contrib_tags.py

```diff
@@ -29,3 +30,13 @@ def concat_strings(*args):
     for string in args:
         concat_str += str(string)
     return concat_str
+
+
+@register.assignment_tag
```

Comment:
For the record: Those tags could probably also be `@register.simple_tags` since they are never used for assignment.